From 64748ed5e45177224699c4f597d81d6cae3bfcdc Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sat, 30 Nov 2013 20:08:16 +0100
Subject: [PATCH] tests: always the system temporary dir

the KDE directory might need further directories in $HOME, which we
don't want to create; hence, force QDir::tempPath() as base directory
for the temporary files
---
 Tests/testcsvparser.cpp | 2 ++
 Tests/testfwparser.cpp  | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/Tests/testcsvparser.cpp b/Tests/testcsvparser.cpp
index e4b8bbc..806e105 100644
--- a/Tests/testcsvparser.cpp
+++ b/Tests/testcsvparser.cpp
@@ -25,6 +25,7 @@
 
 #include "testcsvparser.h"
 
+#include <QDir>
 #include <ktemporaryfile.h>
 
 TestCSVParser::TestCSVParser(): QObject() {
@@ -85,6 +86,7 @@ void TestCSVParser::initTestCase() {
   test_cases_.append(",,,,,,,,,,,\n");
   test_cases_.append("\n");
   KTemporaryFile temp_file;
+  temp_file.setPrefix(QDir::tempPath() + "/");
   temp_file.setSuffix(".txt");
   temp_file.setAutoRemove(false);
   QVERIFY(temp_file.open());
diff --git a/Tests/testfwparser.cpp b/Tests/testfwparser.cpp
index 37f5c48..c30716c 100644
--- a/Tests/testfwparser.cpp
+++ b/Tests/testfwparser.cpp
@@ -17,6 +17,7 @@
 
 #include "testfwparser.h"
 
+#include <QDir>
 #include <ktemporaryfile.h>
 
 TestFWParser::TestFWParser(): QObject() {
@@ -30,6 +31,7 @@ void TestFWParser::initTestCase() {
   test_cases_.append("this is an ex\n\n");
 
   KTemporaryFile temp_file;
+  temp_file.setPrefix(QDir::tempPath() + "/");
   temp_file.setSuffix(".txt");
   temp_file.setAutoRemove(false);
   QVERIFY(temp_file.open());
-- 
1.8.4.2

